src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const splitString = sourceString.split(';');
```

Comment:
Splitting the string by semicolons is a good start, but you should also consider trimming each resulting string to remove any leading or trailing whitespace.